infrastructure/general/setup-bootnode.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Set Up a Bootnode
+description: Learn how to configure and run a bootnode for Polkadot, including P2P, WS, and secure WSS connections with network key management and proxies.
+---
+
+# Set Up a Bootnode
+
+## Introduction
+
+Bootnodes are essential for helping blockchain nodes discover peers and join the network. When a node starts, it needs to find other nodes, and bootnodes provide an initial point of contact. Once connected, a node can expand its peer connections and play its role in the network, like participating as a validator. This guide will walk you through setting up a Polkadot bootnode, configuring P2P, WebSocket (WS), and secure WSS connections, and managing network keys. You’ll also learn how to test your bootnode to ensure it is running correctly and accessible to other nodes.
+
+## Prerequisites
+
+Before you start, you need to have the following prerequisites:
+
+<!-- TODO: What goes here? -->
+
+## Accessing the Bootnode
+
+To connect with other nodes in the network, bootnodes must be accessible through three key channels:
+
+- **P2P** - a direct peer-to-peer connection, set by `--listen-addr /ip4/0.0.0.0/tcp/<port>`. This is not enabled by default on non-validator nodes like archive RPC nodes
+- **P2P/WS** - a WebSocket (WS) connection, also configured via `--listen-addr`
+- **P2P/WSS** - a secure WebSocket (WSS) connection using SSL, often required for light clients. An SSL proxy is needed, as the node itself cannot handle certificates
+
+## Network Key
+
+Starting a node creates its node key in the `chains/<chain>/network/secret_ed25519` file. You can
```

Comment:
I pulled together an introduction but I don't know enough to beef this up a little bit. This section needs a few things:

1. We use Network Key in the heading and call it the node key the rest of the time. I think we need something explaining it's often referred to as both, etc. 
2. Should I already know what a network key is and why I need it at this point in the docs? A couple of sentences like "A network key is used to...." would make this smoother for the reader
3. Please break out CLI command into code fence blocks so they have the copy/paste button